### PR TITLE
Release 23.5

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 About conda-libmamba-solver-feedstock
 =====================================
 
-Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/conda-libmamba-solver-feedstock/blob/main/LICENSE.txt)
+Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/conda-libmamba-solver-conda-forge-feedstock/blob/main/LICENSE.txt)
```
Comment (jaimergp) [inline README.md]:
```suggestion
Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/conda-libmamba-solver-conda-forge-feedstock/blob/main/LICENSE.txt)
```
```suggestion
Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/conda-libmamba-solver-feedstock/blob/main/LICENSE.txt)
```

File: README.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ Current build status
 
 <table><tr><td>All platforms:</td>
     <td>
-      <a href="https://dev.azure.com/conda-forge/feedstock-builds/_build/latest?definitionId=15690&branchName=main">
-        <img src="https://dev.azure.com/conda-forge/feedstock-builds/_apis/build/status/conda-libmamba-solver-feedstock?branchName=main">
+      <a href="https://dev.azure.com/conda-forge/feedstock-builds/_build/latest?definitionId=&branchName=main">
+        <img src="https://dev.azure.com/conda-forge/feedstock-builds/_apis/build/status/conda-libmamba-solver-conda-forge-feedstock?branchName=main">
```
Comment (jaimergp) [inline README.md]:
```suggestion
      <a href="https://dev.azure.com/conda-forge/feedstock-builds/_build/latest?definitionId=15690&branchName=main">
        <img src="https://dev.azure.com/conda-forge/feedstock-builds/_apis/build/status/conda-libmamba-solver-feedstock?branchName=main">
```

**Review by jezdez — APPROVED** [review]

**Comment by jaimergp** [comment]
Needs https://github.com/conda-forge/conda-feedstock/pull/208 (freshly merged) to be live in CDN.

**Pull request merged**